healthcare/api-client/v1/consent/requirements.txt

```diff
@@ -0,0 +1,4 @@
+google-api-python-client==2.40.0
+google-auth-httplib2==0.1.0
+google-auth==2.5.0
+google-cloud==0.34.0
```

Comment:
This is an old version from 2018: https://pypi.org/project/google-cloud/0.34.0/

Reply:
The guidance from that version says to use the product-specific google-cloud-* packages, but the Healthcare API doesn't have one: https://github.com/googleapis/google-cloud-python

Reply:
Can you help me locate the code in this sample that uses `google-cloud`? In other words, what error will we get if we remove `google-cloud` from this requirements file?

Reply:
You're right, it looks like we don't need it. I thought nox would catch that.